pkg/minishift/config/constants.go

```diff
@@ -0,0 +1,22 @@
+/*
+Copyright (C) 2017 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package config
```

Comment:
Should we move this into `pkg/minishift/constants/constants.go`?  This structure can also handle OS specific constants (if any) similar to the structure of `pkg/minikube/constants` folder.

See here https://github.com/minishift/minishift/pull/1243#discussion_r132901496 .

We can have this structure in @praveenkumar  PR and then you can consume it or vice-versa.